_posts/2022-04-14-nearest-fire-stations-map.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Is your house close enough to the nearest fire station?
+author:
+  display_name: Greg Bunce
+  email: gbunce@utah.gov
+date: 2022-04-14 9:26:46
+categories:
+  - Featured
+tags:
+  - sgid
+  - fire
+  - emergency response
+  - network
+---
+
+Living close to a fire station can be noisy, but on the other hand, not living close enough can result in delayed, or even worse, inadequate fire service. The National Fire Protection Association (NFPA) recommends a total response time of [six minutes](https://www.purvis.com/current-state-of-turnout-times/). This includes one minute for dispatch, one minute for turnout (getting out of the station), and four minutes for travel. Ideally, all occupied buildings in the US would be located within this six-minute window, or response zone. This is where Geographic Information Systems (GIS) come into play. GIS can help us visualize these response zones and also identify neighborhoods with inadequate service.
+  
+### The SGID
+{: .text-left}
+  
+In Utah, the best place to begin any GIS project is with the [State Geographic Information Datasource]({% link data/index.html %}#welcome-to-the-state-geographic-information-database-sgid) (SGID). The SGID is hosted by the [Utah Geospatial Resource Center](https://gis.utah.gov/) (UGRC) and contains hundreds of statewide geospatial datasets, many of which are considered framework data that are used in larger government systems such as 911, voting, and taxation. Because of this, data in the SGID is updated frequently and adheres to established [policies and standards]({% link about/policy/sgid/index.md %}#internal-database-standards).
```

Comment:
you can use link index.html here instead of gis.utah.gov

Reply:
makes sense.  thx